docs/docsite/rst/dev_guide/style_guide/preferred_terms.rst

```diff
@@ -0,0 +1,79 @@
+.. _preferred_terms:
+
+Preferred terminology
+=====================
+
+.. contents::
+  :local:
+
+The following is the list of preferred terms for Ansible ecosystem projects.
+
+
+   Ansible
+      Use on all occurrences or ``ansible`` if referring to the CLI command. Preference is to use this to refer to the Ansible community package.
+
+   Ansible Builder
+      Use on first occurrence or ``ansible-builder`` if referring to the CLI command. Subsequent occurrences can be Builder.
+
+   Ansible Collections
+      Use on first occurrence. Subsequent occurrences can be collections.
```

Comment:
Hmm, if C in collections is capitalized in Ansible Collections, it really sounds like a product to me (which implies there is one Ansible Collections thing, and not that there are many Ansible collections).

The same applies to Ansible Execution Environments, Ansible Playbook, Ansible Roles, Ansible Rule, Ansible Rulebook, i.e. basically everything where from the second occurrence the second term is used lower-case.

Reply:
We went back and forth (and back again) on this one.  I think the request stems from https://github.com/ansible/ansible-documentation/issues/152#issuecomment-1660625722

So I tried to split it in the middle-keep the caps for the first full occurrence, and then drop later on for most, unless it's the official project name.

Reply:
Rereading https://github.com/ansible/ansible-documentation/issues/152#issuecomment-1661862668 and will try to apply in an update here. That comment did suggest Ansible collections, not Ansible Collections.